src/common/admin_socket.cc

```diff
@@ -41,6 +42,10 @@
 #include "include/sock_compat.h"
 #include "fmt/format.h"
 
+#ifndef WIN32
```

Comment:
Can you move this system include to before the project ones?

Reply:
Is this a Ceph code style policy? There are good reasons for including system headers last: if you include system headers first, you hide bugs in your own headers when there are `#include` lines missing, because the parent source file already included them.

Reply:
The `CodingStyle` file refers to Google's cppguide https://google.github.io/styleguide/cppguide.html
And indeed Google chose an ordering that I do not understand - they really do want to include system headers before project's headers, but do not explain why.
The ordering of includes in Ceph rarely ever follows these rules, `admin_socket.cc` is no exception - it's chaos.
But I'll try to change my PR according to the Google rules (even if I dislike these rules for good reasons).

Reply:
moved

Reply:
I usually follow that rule, with the one exception:
in file.cc
I am always placing "include file.h" as the first include line.